jetstream/lifestyles-images-onboarding-experiment-v3.toml

```diff
@@ -0,0 +1,54 @@
+## EXPERIMENT SPECIFIC
+[experiment]
+segments = ['attributed', 'non_attributed']
+
+start_date = "2023-06-14"
+
+end_date = "2023-06-27"
+
+reference_branch = "control"
+
+
+
+## SEGMENTS
+# A segment for users that were acquired through ctd creatives
+# first run feature.
+[segments]
+
+[segments.attributed]
+select_expression = 'max(eligible_segment = TRUE)'
+data_source = "lst"
+
+[segments.non_attributed]
+select_expression = 'max(eligible_segment = FALSE)'
+data_source = "lst"
+
+## Data Sources
+[segments.data_sources.lst]
+from_expression = """
+    (SELECT a.*
+      , CASE WHEN b.adjust_network in ("Google Ads ACI", "Product Marketing (Owned media)") THEN TRUE else FALSE END AS eligible_segment
```

Comment:
I'd normally leave the CASE out if the returned value should be TRUE/FALSE.
Writing just `b.adjust_network in ("Google Ads ACI", "Product Marketing (Owned media)") AS eligible_segment` achieves the same result and I think it's more legible